python/tvm/topi/tensor.py

```diff
@@ -73,3 +75,341 @@ def full_like(x, fill_value):
         The result.
     """
     return cpp.full_like(x, fill_value)
+
+
+def segment_max(data, segment_ids, num_out):
```

Comment:
Could you add a bit more comments through this file? This would make it easier to read and also more future proof

Reply:
👌